### PR TITLE
Update Tessian app

### PR DESCRIPTION
**Pull request by alxwrd**
## PR Details

This updates the Tessian app to be compatiable with our new event schema. I've updated the version in the manifest to be `2.0` as these changes will be breaking with old data. I haven't updated a `CHANGELOG.md` because I couldn't find one in the repository. 

## Checklist

- [x] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [ ] Updated CHANGELOG.md.
- [x] Ran unit tests locally.
- [x] Updated screenshots/sample logs if any.

### CODE REVIEW

File: Tessian/Tessian.manifest.json [hunk 1]
```diff
@@ -26,12 +26,11 @@
...
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_overview_dashboard.jpg",
-        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_constructor_dashboard.jpg"
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_overview_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_guardian_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_defender_dashboard.png",
+        "https://sumologic-partner-appsdata.s3.amazonaws.com/dashboards/Tessian/tessian_architect_dashboard.png"
     ],
-    "uuid": "9ddbe84d-75af-482d-b643-c5b04d3ffce5",
-    "version": "1.0"
+    "uuid": "d3ce3ac9-4180-4453-b288-e9d32a015da4",
```
Comment (ankitgoelcmu) [inline Tessian/Tessian.manifest.json:34]: uuid doesn't change when we update the app. it remains same during life cycle of the app. Please revert back to original one

File: Tessian/resources/logs/tessian_logs.txt [hunk 2]
```diff
@@ -1,186 +1,200 @@
...
+{"architect_details": {"breach_prevented": null, "final_outcome": "SENT_WITHOUT_CHANGES", "is_sensitive": false, "justifications": [], "triggered_logic_types": ["KEYWORD_PATTERNS", "REGEX"], "triggered_policy_ids": ["cc25c808-df24-4e21-ac99-2a347e11ae61"], "triggered_policy_names": ["Architect E2E Warn"], "user_responses": ["SEND"], "user_shown_message": true}, "created_at": "2022-09-29T00:06:53.836812Z", "id": "architect::outbound-3338690", "outbound_email_details": {"attachments": {"bytes": 0, "count": 0, "names": []}, "from": "andy@rotepad.com", "message_id": null, "recipients": {"all": ["architect_testing@rotepad.com"], "bcc": [], "cc": [], "count": 1, "to": ["architect_testing@rotepad.com"]}, "reply_to": [], "send_time": "2022-09-29T00:06:53.536451Z", "subject": "Test", "tessian_action": "SILENTLY_TRACK", "tessian_id": "dff6731c-57b7-444b-9eb2-92cbe1e0fff4", "transmitter": "andy@rotepad.com"}, "portal_link": "https://architest.tessian.xyz/0/events/architect?check_log_id=3338690&utm_source=external_api&utm_medium=trigger&utm_campaign=architect", "type": "architect", "updated_at": "2022-09-29T00:06:54.084847Z"}
```
Comment (ankitgoelcmu) [inline Tessian/resources/logs/tessian_logs.txt:1]: I tested the app with these logs but some of the dashboards panels are not loading up due to lack to sample data. Example is Overview/Suspicious Emails Detected

Comment (ankitgoelcmu) [reply Tessian/resources/logs/tessian_logs.txt:1]: Can you provide all logs which will light up all dashboards

**Review by ankitgoelcmu — CHANGES_REQUESTED** [review]

**Review by ankitgoelcmu — APPROVED** [review]
LGTM

**Review by ankitgoelcmu — DISMISSED** [review]

File: Tessian/resources/logs/tessian_logs.txt (hunk 2 above)
Comment (alxwrd) [reply Tessian/resources/logs/tessian_logs.txt:1]: I can look at providing example logs for all dashboards on Monday.

Comment (alxwrd) [reply Tessian/resources/logs/tessian_logs.txt]: [2bd81df](https://github.com/SumoLogic/sumologic-public-partner-apps/pull/55/commits/2bd81df2749b7da330543dcea0b18e663e1ed63f)

**Review by ankitgoelcmu — APPROVED** [review]
LGTM

**Pull request merged**